test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class UserTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
Rails by default uses MiniTest as its test library, but since you added RSpec you should've remove the minitest files.